kubernetes/aks-base/Makefile-impl

```diff
@@ -0,0 +1,67 @@
+
+AKS_BASE_MKFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
+AKS_BASE_MKFILE_DIR := $(dir $(AKS_BASE_MKFILE_PATH))
+
+include $(AKS_BASE_MKFILE_DIR)../common/Makefile
+
+AKS_BASE_CLUSTER_ID ?= cp-examples-operator-$(USER)
+AKS_BASE_REGION ?= centralus
+AKS_BASE_ZONE ?= 1
+AKS_BASE_CLUSTER_VERSION ?= 1.17.13
+AKS_BASE_MACHINE_TYPE ?= Standard_D4s_v4
+AKS_BASE_DISK_SIZE ?= 100
+AKS_BASE_NUM_NODES ?= 3
+
+ZOOKEEPER_REPLICAS ?= 1
+KAFKA_REPLICAS ?= 1
+CONNECT_REPLICAS ?= 1
+SCHEMA_REGISTRY_REPLICAS ?= 1
+
+KUBECTL_CONTEXT ?= aks_$(AZ_RESOURCE_GROUP)_$(AKS_BASE_REGION)_$(AKS_BASE_CLUSTER_ID)
+KUBECTL_NAMESPACE ?= operator
+
+ADDITIONAL_HELM_FLAGS ?=
+
+SKIP_CLICKS ?=
+SKIP_CLIENT_CONSOLE ?=
+ 
+HELM_COMMON_FLAGS := --wait --timeout=5m -f $(AKS_BASE_MKFILE_DIR)cfg/values.yaml --set global.initContainer.image.tag=$(OPERATOR_CP_IMAGE_TAG) --set global.provider.region=$(AKS_BASE_REGION) --set global.provider.kubernetes.deployment.zones={$(AKS_BASE_REGION)-$(AKS_BASE_ZONE)} $(ADDITIONAL_HELM_FLAGS)
+
+aks-check-dependencies: check-dependencies
```

Comment:
this is another offender , **check-dependencies** must go as it is already in the **init** function
@rspurgeon